src/components/fb/CreateFidelityBond.module.css

```diff
@@ -4,6 +4,13 @@
   padding: 1.25rem;
 }
 
+.containerWhenBondAlreadyExists {
+  border: 1px solid var(--bs-gray-200);
+  border-radius: 0.3rem;
+  display: block;
+  margin-left: 41rem;
```

Comment:
Instead of positioning with margins, what do think about using flexbox?
e.g. you could use `d-flex justify-content-end` on the parent element to position the button to the "right".